packages/desktop-client/src/components/autocomplete/FilterList.tsx

```diff
@@ -30,6 +25,7 @@ function FilterList<T extends { id: string; name: string }>({
           ...(!embedded && { maxHeight: 175 }),
         }}
       >
+        <ItemHeader title="Saved Filters" />
```

Comment:
@carkom should we pass a `type` here? Currently we are getting a data-testid here of `Saved-Filters-undefined-item-group` for this component.